### PR TITLE
fix netlify (?)

### LINKED ISSUE #1058: Deploys are failing
```
12:06:30 PM: Build ready to start
12:06:32 PM: Fetching cached dependencies
12:06:32 PM: Starting to download cache of 314.7MB
12:06:34 PM: Finished downloading cache in 2.110743212s
12:06:34 PM: Starting to extract cache
12:06:43 PM: Finished extracting cache in 9.107995592s
12:06:43 PM: Finished fetching cache in 11.444127128s
12:06:43 PM: Starting to prepare the repo for build
12:06:44 PM: Preparing Git Reference refs/heads/master
12:06:45 PM: Starting build script
12:06:45 PM: Installing dependencies
12:06:46 PM: Started restoring cached node version
12:06:48 PM: Finished restoring cached node version
12:06:48 PM: Attempting node version 'lts/carbon' from .nvmrc
12:06:48 PM: Downloading and installing node v8.11.3...
12:06:48 PM: Downloading https://nodejs.org/dist/v8.11.3/node-v8.11.3-linux-x64.tar.xz...
12:06:49 PM: 
##
12:06:49 PM:   3.5%
12:06:49 PM: 
##########
12:06:49 PM:                              15.2%
12:06:49 PM: 
#######################
12:06:49 PM:                                      33.2%
12:06:49 PM: 
###############################
12:06:49 PM: #############                              61.3%
12:06:49 PM: 
######################################################################## 100.0%
12:06:49 PM: Computing checksum with sha256sum
12:06:49 PM: Checksums matched!
12:06:52 PM: Now using node v8.11.3 (npm v5.6.0)
12:06:52 PM: Attempting ruby version 2.2.9, read from environment
12:06:53 PM: Using ruby version 2.2.9
12:06:53 PM: Using PHP version 5.6
12:06:53 PM: Started restoring cached node modules
12:06:53 PM: Finished restoring cached node modules
12:06:53 PM: Started restoring cached yarn cache
12:06:57 PM: Finished restoring cached yarn cache
12:06:57 PM: Found yarn version (0.18.1) that doesn't match expected (1.7.0)
12:06:58 PM: up to date in 0.06s
12:06:58 PM: Installing yarn at version 1.7.0
12:06:58 PM: Installing Yarn!
12:06:58 PM: > Downloading tarball...
12:06:58 PM: [1/2]: https://yarnpkg.com/downloads/1.7.0/yarn-v1.7.0.tar.gz --> /tmp/yarn.tar.gz.A6297dhAXv
12:06:58 PM:   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
12:06:58 PM:                                  Dload  Upload   Total   Spent    Left  Speed
12:06:58 PM: 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
12:06:58 PM: 
100    91  100    91    0     0    716      0 --:--:-- --:--:-- --:--:--   722
12:06:58 PM: 
  0     0    0   608    0     0   1823      0 --:--:-- --:--:-- --:--:--  1823
12:06:58 PM: 
 12  922k   12  118k    0     0   180k      0  0:00:05 --:--:
12:06:58 PM: --  0:00:05  180k
12:06:59 PM: 
100  922k  100  922k    0     0   942k
12:06:59 PM:      0 --:--:-- --:--:-- --:--:-- 2489k
12:06:59 PM: [2/2]: https://yarnpkg.com/downloads/1.7.0/yarn-v1.7.0.tar.gz.asc --> /tmp/yarn.tar.gz.A6297dhAXv.asc
12:06:59 PM: 
100    95  100    95    0     0   4169      0 --:--:-- --:--:-- --:--:--  4169
12:06:59 PM: 
  0     0    0   612    0     0   4599      0 --:--:-- --:--:-- --:--:--  4599
12:06:59 PM: 
100   832  100   832    0     0   3885      0 --:--:-- --:--:-- --:--:--  3885
12:06:59 PM: > Verifying integrity...
12:06:59 PM: gpg: Signature made Wed 23 May 2018 09:02:50 PM UTC using RSA key ID B6FF4DE3
12:06:59 PM: gpg: Good signature from "Yarn Packaging <yarn@dan.cx>"
12:06:59 PM: gpg: WARNING: This key is not certified with a trusted signature!
12:06:59 PM: gpg:          There is no indication that the signature belongs to the owner.
12:06:59 PM: Primary key fingerprint: 72EC F46A 56B4 AD39 C907  BBB7 1646 B01B 86E5 0310
12:06:59 PM:      Subkey fingerprint: E219 30C4 D0A4 AA46 1858  1F7A E074 D16E B6FF 4DE3
12:06:59 PM: > GPG signature looks good
12:07:00 PM: > Extracting to ~/.yarn...
12:07:00 PM: > Adding to $PATH...
12:07:00 PM: > We've added the following to your /opt/buildhome/.profile
12:07:00 PM: > If this isn't the profile of your current shell then please add the following to your correct profile:
12:07:00 PM: export PATH="$HOME/.yarn/bin:$HOME/.config/yarn/global/node_modules/.bin:$PATH"
12:07:00 PM: 
12:07:00 PM: > Successfully installed Yarn 1.7.0! Please open another terminal where the `yarn` command will now be available.
12:07:00 PM: Installing NPM modules using Yarn version 1.7.0
12:07:01 PM: yarn install v1.7.0
12:07:01 PM: [1/5] Validating package.json...
12:07:01 PM: [2/5] Resolving packages...
12:07:01 PM: warning Resolution field "graphql@0.10.5" is incompatible with requested version "graphql@^0.11.3"
12:07:01 PM: warning Resolution field "graphql@0.10.5" is incompatible with requested version "graphql@^0.11.7"
12:07:01 PM: warning Resolution field "react@16.0.0" is incompatible with requested version "react@^15.6.0"
12:07:01 PM: warning Resolution field "react-dom@16.0.0" is incompatible with requested version "react-dom@^15.6.0"
12:07:02 PM: [3/5] Fetching packages...
12:07:21 PM: info fsevents@1.1.3: The platform "linux" is incompatible with this module.
12:07:21 PM: info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
12:07:21 PM: [4/5] Linking dependencies...
12:07:21 PM: warning " > eslint-config-fbjs@2.0.1" has incorrect peer dependency "babel-eslint@^7.2.3".
12:07:21 PM: warning "gatsby > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
12:07:21 PM: warning " > request-promise@4.2.2" has unmet peer dependency "request@^2.34".
12:07:21 PM: warning "request-promise > request-promise-core@1.1.1" has unmet peer dependency "request@^2.34".
12:07:27 PM: [5/5] Building fresh packages...
12:07:27 PM: success Saved lockfile.
12:07:27 PM: Done in 26.24s.
12:07:27 PM: NPM modules installed using Yarn
12:07:28 PM: Started restoring cached go cache
12:07:28 PM: Finished restoring cached go cache
12:07:28 PM: unset GOOS;
12:07:28 PM: unset GOARCH;
12:07:28 PM: export GOROOT='/opt/buildhome/.gimme/versions/go1.10.linux.amd64';
12:07:28 PM: export PATH="/opt/buildhome/.gimme/versions/go1.10.linux.amd64/bin:${PATH}";
12:07:28 PM: go version >&2;
12:07:28 PM: export GIMME_ENV='/opt/buildhome/.gimme/env/go1.10.linux.amd64.env';
12:07:28 PM: go version go1.10 linux/amd64
12:07:28 PM: Installing missing commands
12:07:28 PM: Verify run directory
12:07:28 PM: Executing user command: yarn netlify
12:07:29 PM: yarn run v1.7.0
12:07:29 PM: $ yarn install && yarn build
12:07:29 PM: [1/5] Validating package.json...
12:07:29 PM: [2/5] Resolving packages...
12:07:29 PM: warning Resolution field "graphql@0.10.5" is incompatible with requested version "graphql@^0.11.3"
12:07:29 PM: warning Resolution field "graphql@0.10.5" is incompatible with requested version "graphql@^0.11.7"
12:07:29 PM: warning Resolution field "react@16.0.0" is incompatible with requested version "react@^15.6.0"
12:07:29 PM: warning Resolution field "react-dom@16.0.0" is incompatible with requested version "react-dom@^15.6.0"
12:07:30 PM: [3/5] Fetching packages...
12:07:49 PM: info fsevents@1.1.3: The platform "linux" is incompatible with this module.
12:07:49 PM: info "fsevents@1.1.3" is an optional dependency and failed compatibility check. Excluding it from installation.
12:07:49 PM: [4/5] Linking dependencies...
12:07:49 PM: warning " > eslint-config-fbjs@2.0.1" has incorrect peer dependency "babel-eslint@^7.2.3".
12:07:49 PM: warning "gatsby > postcss-cssnext@2.11.0" has unmet peer dependency "caniuse-db@^1.0.30000652".
12:07:49 PM: warning " > request-promise@4.2.2" has unmet peer dependency "request@^2.34".
12:07:49 PM: warning "request-promise > request-promise-core@1.1.1" has unmet peer dependency "request@^2.34".
12:07:53 PM: [5/5] Building fresh packages...
12:07:54 PM: success Saved lockfile.
12:07:54 PM: $ gatsby build
12:07:56 PM: success delete html files from previous builds — 0.057 s
12:07:56 PM: success open and validate gatsby-config.js — 0.005 s
12:07:56 PM: success copy gatsby files — 0.023 s
12:07:57 PM: error UNHANDLED EXCEPTION
12:07:57 PM: 
12:07:57 PM:   Error: Cannot find module '../build/Release/sharp.node'
12:07:57 PM:   
12:07:57 PM:   - v8-compile-cache.js:159 require
12:07:57 PM:     [repo]/[v8-compile-cache]/v8-compile-cache.js:159:20
12:07:57 PM:   
12:07:57 PM:   - constructor.js:8 Object.<anonymous>
12:07:57 PM:     [repo]/[sharp]/lib/constructor.js:8:15
12:07:57 PM:   
12:07:57 PM:   - v8-compile-cache.js:178 Module._compile
12:07:57 PM:     [repo]/[v8-compile-cache]/v8-compile-cache.js:178:30
12:07:57 PM:   
12:07:57 PM:   - v8-compile-cache.js:159 require
12:07:57 PM:     [repo]/[v8-compile-cache]/v8-compile-cache.js:159:20
12:07:57 PM:   
12:07:57 PM:   - index.js:3 Object.<anonymous>
12:07:58 PM:     [repo]/[sharp]/lib/index.js:3:15
12:07:58 PM:   
12:07:58 PM:   - v8-compile-cache.js:178 Module._compile
12:07:58 PM:     [repo]/[v8-compile-cache]/v8-compile-cache.js:178:30
12:07:58 PM:   
12:07:58 PM:   - v8-compile-cache.js:159 require
12:07:58 PM:     [repo]/[v8-compile-cache]/v8-compile-cache.js:159:20
12:07:58 PM:   
12:07:58 PM:   - index.js:519 Object.<anonymous>
12:07:58 PM:     [repo]/[gatsby-plugin-sharp]/index.js:519:13
12:07:58 PM:   
12:07:58 PM:   - v8-compile-cache.js:178 Module._compile
12:07:58 PM:     [repo]/[v8-compile-cache]/v8-compile-cache.js:178:30
12:07:58 PM:   
12:07:58 PM: 
12:07:58 PM: error Command failed with exit code 1.
12:07:58 PM: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
12:07:58 PM: error Command failed with exit code 1.
12:07:58 PM: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
12:07:59 PM: Caching artifacts
12:07:59 PM: Started saving node modules
12:07:59 PM: Finished saving node modules
12:07:59 PM: Started saving yarn cache
12:08:00 PM: Finished saving yarn cache
12:08:00 PM: Started saving pip cache
12:08:03 PM: Finished saving pip cache
12:08:03 PM: Started saving emacs cask dependencies
12:08:03 PM: Finished saving emacs cask dependencies
12:08:03 PM: Started saving maven dependencies
12:08:03 PM: Finished saving maven dependencies
12:08:03 PM: Started saving boot dependencies
12:08:03 PM: Finished saving boot dependencies
12:08:03 PM: Started saving go dependencies
12:08:03 PM: Finished saving go dependencies
12:08:03 PM: Cached node version v8.11.3
12:08:03 PM: Error running command: Build script returned non-zero exit code: 1
12:08:03 PM: Failing build: Failed to build site
12:08:04 PM: failed during stage 'building site': Build script returned non-zero exit code: 1
12:08:04 PM: Finished processing build request in 1m32.371974109s
```

### PR DESCRIPTION
**Pull request by alexkrolick**
tweaking stuff for netlify to try to fix #1058

### CODE REVIEW

File: src/templates/community.js [hunk 1]
```diff
@@ -35,7 +35,9 @@ export const pageQuery = graphql`
         prev
       }
       fields {
-        path
+        path {
+          id
```
Comment (alexkrolick) [inline src/templates/community.js]: @KyleAMathews any idea why this now requires `{ ... }`?

screen shot 2018-07-16 at 9 23 53 pm

**Comment by alexkrolick** [comment]
new error at least:
```
9:45:41 PM: error Failed to process image /opt/build/repo/content/images/docs/thinking-in-react-tagtree.png
9:45:41 PM: 
9:45:41 PM:   Error: spawn /opt/build/repo/node_modules/pngquant-bin/vendor/pngquant ENOENT
9:45:41 PM:   
9:45:41 PM:   - child_process.js:190 Process.ChildProcess._handle.onexit
9:45:41 PM:     internal/child_process.js:190:19
9:45:41 PM:   
9:45:41 PM:   - child_process.js:372 onErrorNT
9:45:41 PM:     internal/child_process.js:372:16
9:45:41 PM:   
9:45:41 PM:   - next_tick.js:138 _combinedTickCallback
9:45:41 PM:     internal/process/next_tick.js:138:11
9:45:41 PM:   
9:45:41 PM:   - next_tick.js:180 process._tickCallback
9:45:41 PM:     internal/process/next_tick.js:180:9
9:45:41 PM:   
```

**Comment by alexkrolick** [comment]
`(ノ°Д°）ノ︵ ┻━┻`

IDK... you can debug the Netlify docker build locally apparently, might try that later https://github.com/netlify/build-image#testing-locally

**Comment by KyleAMathews** [comment]
Looks like a bad node_modules cache where pngquant is missing

**Comment by alexkrolick** [comment]
Nuking the cache before every build seems reasonable given how many problems we've had with it

**Comment by sophiebits** [comment]
Thank you!

**Pull request merged**